### DISCUSSION TITLE
Get your Reqnroll sticker!

### DISCUSSION THREAD

**Discussion by gasparnagy** [reqnroll/Reqnroll#98] [Contributor Q&A]
I have received the first set of Reqnroll 💻 stickers! @reqnroll/core-team: Please drop a mail to gaspar at specsolutions.eu with your postal address so that I can send you some! 

For mass ordering you can register to Sticker Mule with this [referral link](https://www.stickermule.com/eu/unlock?ref_id=8203070701&utm_medium=link&utm_source=invite) and order from [my profile](https://www.stickermule.com/eu/u/specsolutions).

![reqnroll_stickers](https://github.com/reqnroll/Reqnroll/assets/144701/b0288310-7462-439b-8ddd-6dd78f39b5b7)

**Comment by jacobduijzer** [comment]
Well, I've got one, and it is on my laptop already, thanks!

![reqnroll](https://github.com/reqnroll/Reqnroll/assets/1770444/5950851d-bc1c-42f9-afd5-84c4931b253e)

**Comment by gasparnagy** [comment]
@mbhoek @Socolin @SabotageAndi @FantasyTeddy @clrudolphi @Code-Grump @Tiberriver256 Send me your postal address to gaspar at specsolutions.eu. If you send it by Thursday (25/04) night, I can post them on Friday.
